subsys/nv_config/Kconfig

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig NV_CONFIG_SYSTEM
+	bool
+	default y
+	prompt "Enable nonvolatile configuration system"
+	depends on FILE_SYSTEM || FCB
+	select NEWLIB_LIBC
+	select MBEDTLS
```

Comment:
why mbedtls? does FCB and FS both need mbedtls support?

Reply:
configuration requires base64 encoding which is available in mbedtls. The same we did for mcumgr recently.

Reply:
Ok.